commands/import-item.js

```diff
@@ -43,66 +43,59 @@ module.exports = Command.extend({
         return r;
     },
 
-    options: {
+    options: util.buildOpts({
         target: {type: 'string', alias: 't', default: './'},
-        watch: {type: 'boolean', alias: 'W'},
+        watch: {type: 'boolean', alias: 'w'},
```

Comment:
Is this correct? The prompt at the top of the file says that -w is for --password, and -W for --watch